### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by rex-9**
- Implemented according to the Bookstore design in [Zeplin](https://zeplin.io/).
- To log in to Zeplin, use the following credentials:
  - username: student@microverse.org
  - password: xgJh54DuV8f5gFA
- Use the information detailed in Zeplin to style the website.
- I've implemented my best to match Zeplin's design.

### CODE REVIEW

File: src/components/AddBook.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const AddBook = () => {
 
   return (
     <div id="addBook">
-      <h2>Add Book</h2>
+      <h2>Add New Book</h2>
       <div className="container">
         <input type="text" placeholder="Add Title" value={title} onChange={changeTitle} />
         <input type="text" placeholder="Add Author" value={author} onChange={changeAuthor} />
```
Comment (Ismailco) [inline src/components/AddBook.js:44]: - [ ] [OPTIONAL] Kindly remove all `console.log()` in your code, it makes your code slower

File: src/components/Nav.js [hunk 2]
```diff
@@ -1,15 +1,68 @@
...
+            >
+              Categories
+            </Link>
+          </li>
+        </ul>
+      </div>
+      <img
+        width="30px;"
+        height="30px;"
+        style={{
```
Comment (Ismailco) [inline src/components/Nav.js]: - [ ] If you need to use inline styles, you can create a variable with your styles and then use it in the style tag, (e.g style={styleVariable}), this is a requirement

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @HtetNaing0814, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @HtetNaing0814 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Your linters are passing all the checks
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**